### ISSUE TITLE
rp2040-zero not correctly handled

### ISSUE THREAD

**Issue by LeonHagenaarsKeus** [mu-editor/mu#2260]
### What were you trying to do?

Edit code on a WaveShare rp2040-zero running CircuitPython

### What steps did you take to trigger the issue?

- Open MU editor
- Switch to rp2040 mode
- Restart MU editor
  - MU suggests that it detects a CircuitPython device, asks to switch to CircuitPython mode
- Press OK
  -  MU complains:
![image](https://user-images.githubusercontent.com/7457283/172363314-0553ba16-26c4-4f37-9ce4-d6ba850016d7.png)

### What did you expect to happen?

A working editor in CircuitPython mode on the waveshare rp2040-zero

### What actually happened?

An editor working with a local folder (not the circuitpython device)

### Operating System Version

Windows 11 21H2 build 22000.708

### Mu Version

1.1.1

### Other Info

Using version below as requested in https://github.com/mu-editor/crash-reports/issues/148:
https://github.com/mu-editor/mu/actions/runs/2444128277

### Editor Log

[mu.log](https://github.com/mu-editor/mu/files/8852553/mu.log)

**Comment by carlosperate** [comment]
Hi @LeonHagenaarsKeus,

Thanks for opening the separate GitHub issue.

How does your CircuitPython device mount itself in the operating system? Is the device name `CIRCUITPY`? What drive letter does it have?
What build of CircuitPython is it running?

**Comment by dhalbert** [comment]
It looks like it was detected and then was lost five seconds later:

```
2022-06-07 12:51:39,497 - mu.interface.main:1072(show_confirmation) DEBUG: Detected new CircuitPython board device.
2022-06-07 12:51:39,497 - mu.interface.main:1073(show_confirmation) DEBUG: Would you like to change Mu to the CircuitPython mode?
2022-06-07 12:51:44,649 - mu.modes.base:61(get_default_workspace) INFO: Using workspace C:\Users\Leon Hagenaars-Keus\mu_code from settings file
2022-06-07 12:51:44,651 - mu.interface.main:1044(show_message) DEBUG: Could not find an attached CircuitPython device.
```

**Comment by LeonHagenaarsKeus** [comment]
Hi @dhalbert 

- Device name: PIDOXL
  (I'm creating my own 'ergodox' based keyboard with a rp2040, the L makes KMK recognize it's the left part of a split keyboard)
- Mount point: D:
- boot_out.txt (Which includes the CircuitPython version and board ID, in case it helps):
```
Adafruit CircuitPython 7.3.0 on 2022-05-23; Waveshare RP2040-Zero with rp2040
Board ID:waveshare_rp2040_zero
```

**Comment by carlosperate** [comment]
> * Device name: PIDOXL

Right, that'd be the reason. Mu expects the CircuitPython devices to have `CIRCUITPY` as the volume name.

I'm not too familiar with the general guidelines for making your own CircuitPython devices, is the current recommendation to change the drive volume name? Is it up to be board designer? or maybe there isn't any guidelines in this area?

**Comment by LeonHagenaarsKeus** [comment]
boot_out.txt is generated at each connect, this file does contain the string 'CircuitPython', so maybe it can be used for the same check?
